Contributing.md

```diff
@@ -1,9 +1,64 @@
-I welcome contributions large and small, and I'm happy to help new developers gets started working on public code.
+<h1>Contributing to SC_Salary_Data</h1>
```

Comment:
[Please use Markdown](https://github.github.com/gfm/#example-85) instead of an h1 tag.

Reply:
Will do. 👍